modules/oxide/src/less/skins/ui/tinymce-5/skin.less

```diff
@@ -178,7 +178,7 @@
   .tox-statusbar__wordcount {
     &:hover,
     &:focus {
-      @{statusbar-not-disabled-selector} {
+      &:not(:disabled):not([aria-disabled=true]) {
```

Comment:
You moved the definition of this variable into this file, but aren't using it? What's the story here? Same in `statusbar.less`